hack/test/e2e-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+export DOCKER_ENGINE_GOARCH=${DOCKER_ENGINE_GOARCH:-amd64}
+
+echo "Ensure emptyfs image is loaded"
+bash /ensure-emptyfs.sh
+
+echo "Run integration/container tests"
+cd /tests/integration/container
+./test.main -test.v
+
+echo "Run integration-cli tests (DockerSuite, DockerNetworkSuite)"
+cd /tests/integration-cli
+./test.main -test.v -check.v -check.f "DockerSuite|DockerNetworkSuite"
```

Comment:
Why only these 2 suites? Do the other suites all expect to start new daemons? 

A comment about why it only runs 2 suites would be great.

Reply:
It's simply that those are the suites I've made work for E2E. Combined they represent ~1200 of the ~1600 total tests. There may be some other suites that we can run but some we will never be able to run in this way, namely `DockerDaemonSuite`, `DockerSwarmSuite`, and `DockerPluginSuite`.

I am continuing work to bring up other suites but will make that more clear in the script.